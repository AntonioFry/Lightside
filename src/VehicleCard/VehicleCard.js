import React from 'react'
import { NavLink } from 'react-router-dom';

const VehicleCard = (props) => {
  const { vehicleName, model, vehicleClass, passengers } = props;
  const vehicleCard = (
  <article className='info-card'>
    <header className='info-card-header'>
      <h3>{vehicleName}</h3>
      <button>Favorite</button>
    </header>
    <NavLink to={`/vehicles/${props.id}`} className='vehicle-card-link'>
      <p>Model: <span className='bold'>{model}</span></p>
      <p>Class: <span className='bold'>{vehicleClass}</span></p>
      <p>Passengers: <span className='bold'>{passengers}</span></p>
    </NavLink>
  </article>
  )
  return (
    vehicleCard
  )
}

export default VehicleCard;