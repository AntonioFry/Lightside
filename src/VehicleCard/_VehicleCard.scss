.vehicle-card-link {
  color: black;
  text-decoration: none;
}