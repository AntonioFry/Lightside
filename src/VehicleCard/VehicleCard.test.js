import React from 'react';
import VehicleCard from './VehicleCard/'
import { shallow, mount } from 'enzyme';

describe('VehicleCard', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(
      <VehicleCard 
      vehicleName='car'
      model='cool one'
      vehicleClass='cooooool one'
      passengers='100'
      />
    )
    expect(wrapper).toMatchSnapshot();
  })
});