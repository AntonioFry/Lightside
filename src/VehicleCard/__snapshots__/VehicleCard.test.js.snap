// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VehicleCard should match the snapshot 1`] = `ShallowWrapper {}`;
